index.html

```diff
@@ -2525,72 +2525,79 @@ <h3>
-    with first item the <a href="https://infra.spec.whatwg.org/#strings">INFRA string</a>
-    <code>https://www.w3.org/ns/did/v1</code>, and subsequent items of type 
-    <a href="https://infra.spec.whatwg.org/#strings">INFRA string</a> or 
-    <a href="https://infra.spec.whatwg.org/#maps">INFRA map</a>.
-    <pre class="example">
-      {
-        "@context": [
-          "https://www.w3.org/ns/did/v1",
-          "https://example.com/blockchain-identity/v1"
-        ],
-        ...
-      }
-    </pre>
-    <p class="issue"  data-number="432">The working group is still discussing restrictions on <code>@base</code>, beyond what JSON-LD allows.</p>
-  </li>
-</ul>
-
-
-All members of the
-<code>@context</code> property SHOULD exist in the DID specification
-registries in order to achieve interoperability across different
-representations. 
+            </p>
 
-If a member does not exist in the DID specification
-registries, then the DID Document might not be interoperable across
-representations.
+            <ul>
+              <li>
+The <a href="https://infra.spec.whatwg.org/#strings">INFRA string</a>
+<code>https://www.w3.org/ns/did/v1</code>.
+                <pre class="example">
+{
+  "@context": "https://www.w3.org/ns/did/v1",
+  ...
+}
+                </pre>
+              </li>
+              <li>
+An <a href="https://infra.spec.whatwg.org/#lists">INFRA list</a>,
+with first item the <a href="https://infra.spec.whatwg.org/#strings">INFRA
+string</a> <code>https://www.w3.org/ns/did/v1</code>, and subsequent items of
+type <a href="https://infra.spec.whatwg.org/#strings">INFRA string</a> or
+<a href="https://infra.spec.whatwg.org/#maps">INFRA map</a>.
+                <pre class="example">
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://example.com/blockchain-identity/v1"
+  ],
+  ...
+}
+                </pre>
+                <p class="issue"  data-number="432">
+The working group is still discussing restrictions on <code>@base</code>,
+beyond what JSON-LD allows.
+                </p>
+              </li>
+            </ul>
 
-<p>
-  Producers SHOULD NOT produce DID Documents that 
-  contain properties not defined via the <code>@context</code>. 
-  Properties that are not defined via the <code>@context</code> MAY be dropped by Consumers.
-</p>
+            <p>
+All members of the <code>@context</code> property SHOULD exist in the DID
+specification registries [[DID-SPEC-REGISTRIES]] in order to achieve
+interoperability across different representations. If a member does not exist
+in the DID specification registries, then the <a>DID document</a> might not be
+interoperable across representations.
+            </p>
 
+            <p>
+It is RECOMMENDED that dereferencing each <a>URI</a> value of the
+<code>@context</code> property results in a document containing
+machine-readable information about the context. These URIs SHOULD be
+associated with a cryptographic hash of the content of the JSON-LD Context as
```

Comment:
Do we need to say anything more about how this cryptographic hash should be associated? I'm thinking what sort of format should it be provided as (e.g. hashlink, multiformat, etc) as well as what sorts of restrictions do we want to say about the hash used (e.g. MD5 or sha-1 acceptable?)

These are nitpick details that I wasn't sure if it would be better to put in here or in the did-spec-registries document.

Reply:
I agree, and think this should go in the Registries. See also: https://github.com/w3c/did-spec-registries/pull/146

Reply:
I'm in favor of either specifying a single hash format or removing these recommendations... this is creating complexity which I doubt it adding any value... and of course, you can just use IPFS URLs if you are concerned about this.